src/latex2sympy/latex2sympy.py

```diff
@@ -878,14 +871,14 @@ def get_differential_var(self, d):
         return sympy.Symbol(text, real=True, positive=True)
 
     def get_differential_var_str(self, text):
-        for i in range(1, len(text)):  # pragma: no cover - loop break not recognized correctly
+        for i in range(15, len(text)):  # pragma: no cover - loop break not recognized correctly
```

Comment:
@builtbywill 1 -> 15 on purpose?

Reply:
or is that from the num of characters is \\differentialD?

Reply:
@lily514 yeah that was the reason for the change, but looking at the coverage results, doesn't look like the method is called right now 🤔 i'm gonna take another glance at convert_frac and handle_integral and see if i can clean this up a smidge

Reply:
@lily514 updated!